ui/lib/kv/addon/templates/secret/metadata/index.hbs

```diff
@@ -1 +1,6 @@
-<Page::Secret::MetadataDetails @model={{this.model}} @breadcrumbs={{this.breadcrumbs}} />
\ No newline at end of file
+<Page::Secret::MetadataDetails
+  @model={{this.model.metadata}}
+  @path={{this.model.path}}
+  @breadcrumbs={{this.breadcrumbs}}
+  @secretCustomMetadata={{this.model.secret.customMetadata}}
```

Comment:
to match the other details page I think updating this to just `@secret` would give more indication of where this is coming from (and that it's on the secret, not metadata) 
```suggestion
  @secret={{this.model.secret}}
```